README.md

```diff
@@ -32,6 +32,45 @@ The token format is as follows:
 
     <omero_session_key>@<host>:<port>
 
+## Example usage
+
+### Bash:
+```bash
+if omero_user_token get; then
+    echo "Found valid token\n"
+    token=$(omero_user_token get)
+    key=$(echo "${token}" | cut -d "@" -f 1)
+    server_details=$(echo "${token}" | cut -d "@" -f 2)
+    host=$(echo "${server_details}" | cut -d ":" -f 1)
+    port=$(echo "${server_details}" | cut -d ":" -f 2)
+    echo "Connecting to ${host}:${port} with key ${key}"
+else
+    echo "No valid token found"
+    exit -1
+fi
+```
+
+### Python:
+```python
+import subprocess
```

Comment:
Maybe just use the Python API?

```
from omero_user_token import getter
token = getter()
if token is not None:
    omero_session_key = token[:token.find('@')]
    host, port = token[token.find('@') + 1:].split(':')
    ...
```

Or maybe we just add a `login()` that does this for you and either gives you a logged in client or `None`?

Reply:
The problem with `token = getter()` is that it uses sys.exit() for all the token invalid conditions and ends with a spectacular stack trace. No ideal at the moment.

`login()` approach seems to me a better way to handle it.

Reply:
Agreed. Do you want to try and refactor it here or want me to do it? It's not a big change.

Reply:
Docs updated.